tests/checks/mock/test_splunk_topology.py

```diff
@@ -0,0 +1,362 @@
+# stdlib
```

Comment:
we can use the following:
{{baseurl}}/services/search/jobs/admin__admin__search__components_at_1488957449_208/results?output_mode=json&count=1&offset=40003a
```
{
  "messages": [
    {
      "type": "FATAL",
      "text": "Invalid offset."
    }
  ]
}
```